packages/docs/components/HeaderBanner.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <transition name="banner">
+  <Transition name="banner">
```

Comment:
I have tried to check myself whether 'Transition` with capital `T` and `Component` with capital `C` works under this change. But I am not very sure this does work.